### ISSUE TITLE
Incorrect Damage Multiplier on Sneed in the deadmines

### ISSUE THREAD

**Issue by albeecode** [cmangos/wotlk-db#296]
Hello,

I believe I found an issue in the WOTLK-DB with the damage multiplier applied to Sneed the NPC that comes from the shreader in the deadmines.

I compare the values from the classic DB, as well as the WOTLK DB.  The results are below.

Also, as I play through on my private server, should I make a new issue for each instance similar to this that I find, or should I collect them all and submit only once?

I plan on doing a compare / DIFF of the creatures in the DB between classic and WOTLK to try and find other things such as this.  

From my perspective the classic multiplier seems more accurate, as when I just attempted this run in WOTLK version, he was basically 2 shotting my tank, granted not very geared, but a level 18 pally with like 600 HP.

I found this information by running the following SQL commands:
SQL CMD: `select name,minmeleedmg,maxmeleedmg,damagemultiplier from wotlkmangos.creature_template where name = 'Sneed';`
SQL Result: 
```
+-------+-------------+-------------+------------------+
| name  | minmeleedmg | maxmeleedmg | damagemultiplier |
+-------+-------------+-------------+------------------+
| Sneed |          29 |          39 |              7.5 |
+-------+-------------+-------------+------------------+
```

SQL CMD: `select name,minmeleedmg,maxmeleedmg,damagemultiplier from classicmangos.creature_template where name = 'Sneed';`
SQL Result:
```
+-------+-------------+-------------+------------------+
| name  | minmeleedmg | maxmeleedmg | damagemultiplier |
+-------+-------------+-------------+------------------+
| Sneed |          29 |          39 |              3.3 |
+-------+-------------+-------------+------------------+
```

**Comment by Grz3s** [comment]
Thx for your report:
Yes 7.5 is to hight ..
.... Just tried to calculate multiplier using values we have... and this is what i got:'`MIN MELEE 

> 'DAMAGE = (((Creature_Template_Classlevelstats.BaseDamageExp? * Creature_Template.DamageVariance) +
> Creature_Template_Classlevelstats.BaseMeleeAttackPower / 14)) * (Creature_Template.BaseMeleeAttackTime / 1000)) * Creature_Template.Dmg_Multiplier
> 
> MAX MELEE DAMAGE = ((((Creature_Template_Classlevelstats.BaseDamageExp? * Creature_Template.DamageVariance) * 1.5) + 
> (Creature_Template_Classlevelstats.BaseMeleeAttackPower / 14)) * (Creature_Template.BaseMeleeAttackTime / 1000)) * Creature_Template.Dmg_Multiplier'

14,6823 * x = 29   == 1,9751673784080150930031398350395‬ multiplier for min
19,52345 * x = 39  == 1.9975977606416898652645920674881 multiplier for max

@Tobschinski @cala could you pls show where your values comming from? ... 
multiplier 3.3 must come from somewhere :)

**Comment by Tobschinski** [comment]
Bestiary lists Sneed with a range of 97 - 129 damage. He attacks once every 2 seconds (unconfirmed). Given the formula above ...

97 = x * ((9.6823) + (70 / 14)) * (2000 / 1000)
129 = x * ((9.6823 * 1.5) + (70 / 14)) * (2000 / 1000)

x = 3.3

**Comment by Grz3s** [comment]
ok thx
